src/generate.ts

```diff
@@ -27,15 +28,16 @@ export function setupStaticGeneration (nuxt: any, options: ModuleOptions) {
 
   nuxt.hook('generate:done', async () => {
     const { dir: generateDir } = nuxt.options.generate
+    const limit = pLimit(5)
```

Comment:
It is not clear to me without reproduction at what concurrency issue happens but I think we should be safe setting it to 8 or 16.